appveyor.yml

```diff
@@ -1,55 +1,2 @@
 build: false
-platform:
-  - x64
-clone_folder: c:\projects\workspace
-
-environment:
-  matrix:
-  - dependencies: highest
-    php_ver_target: 7.4.11
-    xdebug_ver: '2.9.2-7.4-vc15'
-  - dependencies: current
-    php_ver_target: 7.4.11
-    xdebug_ver: '2.9.2-7.4-vc15'
-
-cache: # cache is cleared when linked file is modified
-    - '%LOCALAPPDATA%\Composer\files -> composer.lock'
-    - composer.phar
-    - C:\ProgramData\chocolatey\bin -> appveyor.yml
-    - C:\ProgramData\chocolatey\lib -> appveyor.yml
-    - C:\tools\php -> appveyor.yml
-
-init:
-    - SET PATH=C:\Program Files\OpenSSL;c:\tools\php;%PATH%
-    - SET ANSICON=121x90 (121x90)
-    - SET COMPOSER_NO_INTERACTION=1
-    - SET COMPOSER_FLAGS=--no-interaction --no-progress --no-suggest --prefer-dist
-
-install:
-    # Enable Windows update service
-    - ps: Set-Service wuauserv -StartupType Manual
-    # Install PHP
-    - choco search php --exact --all-versions -r
-    - ps: appveyor-retry cinst --params '""/InstallDir:C:\tools\php""' --ignore-checksums -y php --version $env:php_ver_target
-    - cd c:\tools\php
-    - copy php.ini-production php.ini /Y
-    - echo date.timezone="UTC" >> php.ini
-    - echo extension_dir=ext >> php.ini
-    - echo extension=php_openssl.dll >> php.ini
-    - echo extension=php_mbstring.dll >> php.ini
-    - echo memory_limit=512M >> php.ini
-    # Install composer and update per matrix
-    - cd c:\projects\workspace
-    - php -r "readfile('http://getcomposer.org/installer');" | php
-    - IF %dependencies%==current appveyor-retry php composer.phar install %COMPOSER_FLAGS%
-    - IF %dependencies%==highest appveyor-retry php composer.phar update %COMPOSER_FLAGS%
-    # Install Xdebug for code coverage
-    - ps: .ci\appveyor_install_xdebug.ps1
-
-test_script:
-    - cd c:\projects\workspace
-    - vendor\bin\phpunit --prepend=tests/xdebug-filter.php --coverage-clover=clover.xml --coverage-xml=coverage/coverage-xml --log-junit=coverage/junit.xml
-    - php bin\infection --threads=%NUMBER_OF_PROCESSORS% --log-verbosity=none --coverage=coverage --no-progress --only-covered --ignore-msi-with-no-mutations
-
-matrix:
-  fast_finish: true
+clone_depth: 1
```

Comment:
why do we need to keep this file?

Reply:
AppVeyor will fail the build without it.

Reply:
so will we remove once 0.18 is not maintained anymore?

Reply:
Yes, that's the idea. Remove it once we don't need it anywhere.